File: src/main/java/de/maggiwuerze/xdccloader/events/bot/TargetBotDeleteEvent.java
package de.maggiwuerze.xdccloader.events.bot;

import de.maggiwuerze.xdccloader.events.EntityDeleteEvent;
import de.maggiwuerze.xdccloader.model.entity.Bot;

public class TargetBotDeleteEvent extends EntityDeleteEvent<Bot> {

	public TargetBotDeleteEvent(Object source, Bot bot) {
		super(source, bot);
	}
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/bot/TargetBotDeleteEvent.kt
package de.maggiwuerze.xdccwebloader.events.bot

import de.maggiwuerze.xdccwebloader.events.EntityDeleteEvent
import de.maggiwuerze.xdccwebloader.model.entity.Bot

class TargetBotDeleteEvent(source: Any, bot: Bot) : EntityDeleteEvent<Bot>(source, bot)